clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -73,6 +73,9 @@
   public FSDataOutputStream append(Path cPath, int bufferSize, Progressable progress)
       throws IOException {
     LOG.info("append(" + cPath + ", " + bufferSize + ", " + progress + ")");
+    if (statistics != null) {
+      statistics.incrementWriteOps(1);
+    }
```

Comment:
The naming here is kind of weird since we are using the parent class's variable, could we just make another member variable `mStatistics = statistics`?